dev.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#1. Bundle
+
+pyinstaller main.py --clean --noconfirm \
+     --additional-hooks-dir=hooks
+
+#2. Go into dist dir
+pushd dist/main
+
+
+#3. Run
+LD_LIBRARY_PATH=torch/lib/ ./main ../../images_fun_examples
```

Comment:
it's unclear for me what these values mean (after a while of looking i suspect that project dir and path to python executable?) --- can you extract them to variables to make this easy to understand at a glance?